vertx-web/src/main/java/io/vertx/ext/web/handler/impl/TimeoutHandlerImpl.java

```diff
@@ -35,8 +35,8 @@ public TimeoutHandlerImpl(long timeout) {
   @Override
   public void handle(RoutingContext ctx) {
 
-    // We send a 408 response after timeout
-    long tid = ctx.vertx().setTimer(timeout, t -> ctx.fail(408));
+    // We send a 503 response after timeout
+    long tid = ctx.vertx().setTimer(timeout, t -> ctx.fail(503));
```

Comment:
That's a somehow a breaking change if an application use this handler and expect 408. It should be listed.